### PR TITLE
Add slider section

### PR DESCRIPTION
**Pull request by MxWoodie**
Add slider section with pagination and arrows
Change variable names to appropriate

### CODE REVIEW

**Comment by angelinabylitskaya** [comment]
image
Please, remove padding from buttons class to make them reusable.

**Comment by angelinabylitskaya** [comment]
Please, add:
- hover on menu items;
- slider arrows;
- active slider pagination state and styles.
Consider changing h2 to h1 in slider section as the main title.

File: index.html [hunk 1]
```diff
@@ -25,7 +24,22 @@
 			</nav>
 		</header>
 		<main>
-			<section>Slider</section>
+			<section class="slider-block">
+				<div class="slider-container">
+					<h2 class="slider-header">Expire</h2>
+					<svg width="200px" height="70px" viewBox="0 0 100 10">
+							<polyline class="polyline" points="0 0, 48 0, 52 4, 110 4" />
```
Comment (angelinabylitskaya) [inline index.html]: Wrong alignment

File: index.html [hunk 2]
```diff
@@ -25,7 +24,22 @@
...
+				<div class="slider-container">
+					<h2 class="slider-header">Expire</h2>
+					<svg width="200px" height="70px" viewBox="0 0 100 10">
+							<polyline class="polyline" points="0 0, 48 0, 52 4, 110 4" />	
+					</svg>
+					<p class="slider-description">Professionaly designed, carefully made for your enjoyement</p>
+					<a class="link-button" href="">explore</a>
+					<a class="link-button" href="">learn more</a>
+				</div>
+				<div class="pagination-container">
```
Comment (angelinabylitskaya) [inline index.html]: Pagination should be a list

File: scss/components/_slider.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
+.slider-block {
+  background-color: $color-background-primary;
+}
+
+.slider-container {
+  padding-left: 100px;
+  padding-right: 100px;
+  
+  .slider-header {
+    font-family: $font-primary-semibold, $font-primary-regular, $font-secondary;
```
Comment (angelinabylitskaya) [inline scss/components/_slider.scss]: Consider creating a mixin for fornt definitions or move text styles to the typography file

Comment (angelinabylitskaya) [inline scss/components/_slider.scss]: Consider creating a mixin for fonts definitions or move text styles to the typography file

File: scss/components/_slider.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  }
+}
+
+.polyline {
+  stroke: $color-text-light; 
+  stroke-width: 0.5; 
+  fill: none;
+}
+
+.link-button {
```
Comment (angelinabylitskaya) [inline scss/components/_slider.scss]: Buttons styles can be reused. Move buttons styles to the separate file

File: scss/_variables.scss [hunk 5]
```diff
@@ -1,9 +1,11 @@
-$color-primary: #fff000;
-$mako: #3D4148;
-$alabaster: #FAFAFA;
-$tamarillo: #902616;
-$red-orange: #FE3C1F;
+$color-background-primary: #3D4148;
```
Comment (angelinabylitskaya) [inline scss/_variables.scss]: Use lower case fro colors

File: scss/components/_slider.scss [hunk 6]
```diff
@@ -1,94 +1,76 @@
 .slider-block {
   background-color: $color-background-primary;
+  max-height: 783px;
```
Comment (ElizavetaSelivanova) [inline scss/components/_slider.scss]: content could be changed in the slider, you shouldn't use fixed max-height (min-height -> it's ok)

File: scss/components/_layout.scss [hunk 7]
```diff
@@ -0,0 +1,4 @@
+body {
+  max-width: 1440px;
```
Comment (kobweb) [inline scss/components/_layout.scss]: You shouldn't restrict body's width, it's better to use some wrapper container instead

File: scss/components/_slider.scss [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+  .slider-button {
+    margin-left: 5px;
+  }
+
+  .slider-button + .slider-button {
+    margin-left: 15px;
+  }
+
+  .arrow.slider {
+    top: 6.5em;
```
Comment (kobweb) [inline scss/components/_slider.scss]: For what you use `em` here?

File: scss/components/_slider.scss [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+  stroke-width: 0.5;
+  fill: none;
+}
+
+.pagination-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 140px auto 0;
+  max-width: 105px;
+  max-height: 25px;
```
Comment (kobweb) [inline scss/components/_slider.scss]: For what `max-height` here?

File: scss/style.scss [hunk 10]
```diff
@@ -2,9 +2,13 @@
 
 @import "fonts";
 @import "variables";
+@import "mixins/fonts";
 @import "typography";
 
 // import components here
 
+@import "components/layout";
```
[the remark this answers is not in the document]
Comment (angelinabylitskaya) [reply scss/style.scss]: Or just in scss folder

File: scss/components/_slider.scss [hunk 11]
```diff
@@ -1,94 +1,76 @@
...
-    padding-top: 140px;
-    color: $color-text-light;
+    letter-spacing: 1px;
+    color: $color-text-white;
   }
 
   .slider-description {
-    font-family: $font-primary-light, $font-primary-regular, $font-secondary;
+    @include font-light;
+    max-width: 960px;
```
[the remark this answers is not in the document]
Comment (angelinabylitskaya) [reply scss/components/_slider.scss]: Agree

**Pull request merged**